client/app/components/login/login.html

```diff
@@ -3,24 +3,25 @@
   <div flex="nogrow" layout="column">
     <div flex id="loginContent" layout-padding class="md-whiteframe-2dp">
       <form login-form ng-submit="$ctrl.login()">
-        <h2>Login</h2>
+        <h2 translate>LOGIN.TITLE</h2>
```

Comment:
yes, that's the other option. I prefer this one.. it's not bad if the user sees this.. and it's a bit less to type ;)